packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1588,11 +1588,24 @@ export class DataViewExpander {
         }
 
         const result = await this.transport.relationships.getRelationshipByAddress({ address });
-        if (result.isError) {
-            return this.expandUnknown(address);
+        if (result.isSuccess) {
+            return await this.expandRelationshipDTO(result.value);
         }
 
-        return await this.expandRelationshipDTO(result.value);
+        const requestResult = (
+            await this.consumption.incomingRequests.getRequests({
+                query: {
+                    peer: address,
+                    status: [LocalRequestStatus.ManualDecisionRequired, LocalRequestStatus.DecisionRequired]
+                }
+            })
+        ).value;
+        requestResult.sort((request1, request2) => (request2.createdAt > request1.createdAt ? 1 : -1));
+        if (requestResult.length > 0) {
+            return this.expandAddressFromRequest(requestResult.at(-1)!);
+        }
```

Comment:
we know there is no relationship - there will only be 0 or 1 request available.

Reply:
it is for the case that multiple different templates have been loaded previously, each generating their own request

Reply:
nope, we block these other loads.

Reply:
```
await exchangeTemplate(sRuntimeServices.transport, rRuntimeServices.transport, templateContent);
await exchangeTemplate(sRuntimeServices.transport, rRuntimeServices.transport, templateContent);
const requestsResult = await rRuntimeServices.consumption.incomingRequests.getRequests({});
```
(tried it in the RequestModule.test around line 100) results in a length 2 array

Reply:
this is **really** bad. I'll look into it.

Reply:
To not have this conversation open forever: I assume now that there is max 1 request.